astro/src/content/docs/apis/users.mdx

```diff
@@ -813,15 +814,21 @@ If FusionAuth sends an email to the user, it will either use the `email` top lev
 
     It is recommended to omit this parameter and allow FusionAuth to generate the identifier. Use this parameter only if you must supply your own value for integration into existing systems.
   </APIField>
-  <APIField name="loginId" type="String" required>
-    The login identifier of the user. The login identifier can be either the `email` or the `username`. The username is not case sensitive.
-  </APIField>
-  <APIField name="sendForgotPasswordEmail" type="Boolean" optional defaults="true">
+  <LoginIdField hide_canonical_details login_id_type_field="loginIdTypes" required />
+  <LoginIdTypeField plural optional since="1.99.9" />
+  <APIField name="sendForgotPasswordEmail" type="Boolean" optional defaults="true" deprecated>
```

Comment:
I was about to say you should use `DeprecatedSince` here but of course you did it!